addons/l10n_mu_account/views/report_invoice.xml

```diff
@@ -17,7 +18,25 @@
                 <span t-if="o.name != '/'" t-field="o.name"/>
             </h2>
         </xpath>
+
+        <xpath expr="//div[@name='address_not_same_as_shipping']//t[@t-set='address']" position="inside">
+            <span t-field="o.partner_id.company_registry"/>
+        </xpath>
+        <xpath expr="//div[@name='address_same_as_shipping']//t[@t-set='address']" position="inside">
+            <span t-field="o.partner_id.company_registry"/>
+        </xpath>
+        <xpath expr="//div[@name='no_shipping']//t[@t-set='address']" position="inside">
+            <span t-field="o.partner_id.company_registry"/>
+        </xpath>
+
+        <xpath expr="//div[@id='total']" position="after">
```

Comment:
just to be sure, I see that the exchange rate is displayed between the 2 tables of VAT when using foreign currency. Is it intended ? In the excalidraw, it seems it was requested after. But I think it looks good as it is now, just want to make sure